# Refining-dual-RNA-seq-mapping
**Title:** Refining dual RNA-seq mapping: sequential and combined approaches in host-parasitic plant dynamics

![pipeline](https://github.com/user-attachments/assets/2d6c2500-47c3-437f-8758-ebb54edba68f)

**Summary:**  
This study evaluates the application of dual RNA-seq to investigate the interactions between parasitic and host plants. 
More specifically, we merged host plant with parasitic plant transcriptomes and compared two alignment approaches:

- **Sequential Approach:** Reads are mapped in two steps, first to the host genome and then to the parasite genome (or vice versa).
- **Combined Approach:** Reads are mapped to a concatenated genome (host + parasite) in a single step.

The results show a high mapping rate (around 90%) and minimal cross-mapping, confirming the effectiveness of the dual RNA-seq approach in accurately separating transcripts from the two organisms within the same taxonomic kingdom. the evaluation metrics (i.e., precision, sensitivity, accuracy, and specificity) were all close to one.

**Objective:**  
To demonstrate that dual RNA-seq analysis, through both sequential and combined approaches, is a valid strategy to study host plant - parasite plant interactions, while minimizing errors due to cross-mapping.

More info can be found at [Article](doi.org/10.3389/fpls.2024.1483717)
